bubblegum/program/src/lib.rs

```diff
@@ -905,15 +920,15 @@ pub mod bubblegum {
         });
         let authority_pda_signer = &[&seeds[..]];
         let cpi_ctx = CpiContext::new_with_signer(
-            ctx.accounts.gummyroll_program.to_account_info(),
-            gummyroll::cpi::accounts::Initialize {
+            ctx.accounts.compression_program.to_account_info(),
+            spl_compression::cpi::accounts::Initialize {
                 authority: ctx.accounts.authority.to_account_info(),
-                merkle_roll: merkle_slab,
-                candy_wrapper: ctx.accounts.candy_wrapper.to_account_info(),
+                merkle_tree: merkle_tree,
+                log_wrapper: ctx.accounts.candy_wrapper.to_account_info(),
             },
             authority_pda_signer,
         );
-        gummyroll::cpi::init_empty_gummyroll(cpi_ctx, max_depth, max_buffer_size)
+        spl_compression::cpi::init_empty_merkle_tree(cpi_ctx, max_depth, max_buffer_size)
```

Comment:
Do you know what was the difference between a gummyroll and a merkle_slab?  Are they all fine being called a merkle_tree because in this PR we are changing `init_empty_gummyroll` to `init_empty_merkle_tree`, and `merkle_slab` to `merkle_tree`.  I don't want to incorrectly overload the term but I don't know the detail.